Source/WebCore/Modules/credentialmanagement/CredentialsContainer.cpp

```diff
@@ -32,8 +32,10 @@
 #include "AbortSignal.h"
 #include "CredentialCreationOptions.h"
 #include "CredentialRequestOptions.h"
+#include "DigitalCredential.h"
 #include "Document.h"
 #include "ExceptionOr.h"
+#include "IdentityRequestOptions.h"
 #include "JSDOMPromiseDeferred.h"
```

Comment:
I suspect you have to include JSDigitialCredential.h here to address the build error. (Or something to that effect in one of these files.)

Reply:
Amazing! thank you.